google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/models/RowMutation.java

```diff
@@ -214,4 +215,18 @@ public MutateRowsRequest toBulkProto(RequestContext requestContext) {
             Entry.newBuilder().setRowKey(key).addAllMutations(mutation.getMutations()).build())
         .build();
   }
+
+  /**
+   * Wraps the protobuf {@link MutateRowRequest}.
+   *
+   * <p>WARNING: Please note that the table name will be overwritten by the configuration in the
+   * BigtableDataClient. The mutation must always be idempotent because it would be retried.
```

Comment:
Please replace with:

```
<p>Please ensure that the MutateRowRequest does not use server side timestamps. The BigtableDataClient assumes that RowMutations are idempotent and is configured to enable retries by default. If serverside timestamps are enabled, this can lead to duplicate mutations
```